api/handlers_spaces.go

```diff
@@ -37,15 +48,68 @@ func (s *server) SpaceGetHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	log.Debugf("found cost explorer result cache %+v", *resultCache)
 
-	spaceID := vars["space"]
-	log.Debugf("getting costs for space %s", spaceID)
+	// vars for checking input times parse and are valid
+	timeValidity := false
+	var scleansedTime string
+	var ecleansedTime string
```

Comment:
lets call these `start` and `end`?

Reply:
done.  Had to wrangle time.Time into string afterall.